### ISSUE TITLE
Mac: RTP effects lose focus (and may, but only may, go behind project window after being applied)

### ISSUE THREAD

**Issue by petersampsonaudacity** [audacity/audacity#1563]
**Describe the bug**
Mac: RTP effects lose focus (and may, but only may, go behind project window after being applied)

Having the focus behave as expected is important for keyboard users. For them, it's not a minor issue.

Regression on 2.1.1.

Originally logged on Bugzilla as P3 1471
https://bugzilla.audacityteam.org/show_bug.cgi?id=1471

**To Reproduce**
1. Add some audio.
2. Open a built-in RTP effect such as Wahwah, or an AU effect or LV2 effect. It does not matter if the third-party effect uses graphical interface or not.  
3. Hit ENTER on keyboard, or click "Apply" to apply the effect. 
4. Observe: The effect applies but loses focus. 

5. ALT + F6 to bring the effect back to focus.
6. COMMAND-TAB out of and back into Audacity. Hit ENTER or click "Apply". 

7. Observe: The effect disappears from view because it has been lowered behind the project window, but is still running 

8. COMMAND + backtick (or ALT + F6) will bring the effect window back. 
9. Observe Given you can only run one RTP effect at a time, all the RTP effects will now appear greyed out (the built-in ones may not be greyed due to bug 1460).

You can also do Steps 1 and 2 then jump to step 4 and the bug will still occur.

**Expected behavior**
The RTP effecy should
a) not lose focus
b) not go behind project window

**Screenshots**

**Additional information (please complete the following information):**
 - OS: macOS
 - Version Audacity 2.1.3 - 3.0.4

**Additional context**
Originally reported as a 3.1.3 bug five years ago  - and no attempt at a fix sinc the 

Testing on macOS 10.15.6 with Audacity 3.0.0 25ddfc7
Focus is still lost - but the effects no longer appear to disappear behind the Audacity window

**Comment by dozzzzer** [comment]
I cannot reproduce this in version 3.3.2
@petersampsonaudacity could you please re-check if that works for you as well?

**Comment by petersampsonaudacity** [comment]
@dozzzzer 

testing on macOS 12.6.3 Monterey with 3.3.2

I can no longer reproduce this when testing with any AU effect.
